vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -16562,7 +16562,8 @@ unit:MOL-PER-M2-DAY
   qudt:symbol "mol/(m²⋅day)" ;
   qudt:ucumCode "mol.m-2.d-1"^^qudt:UCUMcs ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
-  rdfs:label "Moles per square metre per day" ;
+  rdfs:label "Moles per Square Metre per Day"@en ;
+  rdfs:label "Moles per Square Metre per Day"@en-us ;
```

Comment:
Change "Metre" to "Meter"

Reply:
That was my intent 🤦‍♂️ I'll fix that now